src/resources/mathjax_preview.js

```diff
@@ -14,7 +14,7 @@ if (typeof VPlantUMLServer == 'undefined') {
 }
 
 new QWebChannel(qt.webChannelTransport,
-    function(channel) {
+    function (channel) {
```

Comment:
Could you revert all these **style** changes, which makes the PR a mess.

Reply:
I see. I formatted it.